packages/ibm-products/src/components/ProductiveCard/ProductiveCard.stories.js

```diff
@@ -205,6 +208,93 @@ Clickable.args = {
   actionIcons: [],
 };
 
+const stackedCardTiles = [
+  {
+    title: 'Title of the row item 1',
+    users: ['MMax König', 'Joe Watson'],
+  },
+  {
+    title: 'Title of the row item 2',
+    users: ['Burak Karaso', 'Alicia Bertrand'],
+  },
+  {
+    title: 'Title of the row item 3',
+    users: ['Rania Lauwers', 'Jasmin Fischer'],
+  },
+];
+
+const StackedTiles = (opts) => {
+  const { onClick, onKeyDown } = opts;
+  const today = new Date();
+  const weekday = [
+    'Sunday',
+    'Monday',
+    'Tuesday',
+    'Wednesday',
+    'Thursday',
+    'Friday',
+    'Saturday',
+  ];
+  function formatAMPM(date) {
+    var hours = date.getHours();
+    var minutes = date.getMinutes();
+    var ampm = hours >= 12 ? 'PM' : 'AM';
+    hours = hours % 12;
+    hours = hours ? hours : 12;
+    minutes = minutes < 10 ? '0' + minutes : minutes;
+    var strTime = hours + ':' + minutes + ' ' + ampm;
+    return strTime;
+  }
+  return (
+    <>
+      {stackedCardTiles.map((tile, idx) => (
```

Comment:
i think we need to change this implementation slightly.

since this is a feature the component is offering we shouldn't be asking the user to have to contribute any of the `blockClass` elements. what we should do here is instead of passing this content to `children` we should have a prop that accepts an array of items that we can iterate on and pass the class name to directly.

so instead of passing this to `children` lets add a new prop like `stackedItems`. and then something like `stackedItems={[Item, Item2, Item])` and then the user would pass whatever content and structure they want.